docs/pages/transactions/send-batch-transactions/client.mdx

```diff
@@ -0,0 +1,162 @@
+<Info>Required SDK version: ^v4.59.1</Info>
+
+See the [`sendCalls` SDK
+reference](/docs/wallets/reference/account-kit/wallet-client/functions/sendCalls)
+for full descriptions of the parameters used in the following example.
+
+You can batch transactions using either the smart wallet client `sendCalls` or `prepareCalls` actions.
+
+<CodeBlocks>
+
+```ts twoslash title="sendCalls.ts"
+import { Address, erc20Abi, encodeFunctionData, parseEther } from "viem";
+import { client } from "./client";
+import { swapAbi } from "./swapAbi";
+
+const DEMO_USDC_ADDRESS: Address = "0xCFf7C6dA719408113DFcb5e36182c6d5aa491443";
```

Comment:
These are just random addresses for the example yeah? So this example should work fine cause we're just calling empty addresses with random data?

Reply:
Is Sepolia the right chain for this? looks empty to me: https://sepolia.etherscan.io/address/0xCFf7C6dA719408113DFcb5e36182c6d5aa491443